storage/badger/operation/prefix.go

```diff
@@ -36,18 +36,18 @@ const (
 	codeSealedRootHeight        = 27 // the height of the highest sealed block contained in the root snapshot
 
 	// codes for single entity storage
-	// 31 was used for identities before epochs
 	codeHeader               = 30
+	_                        = 31 // DEPRECATED: 31 was used for identities before epochs
 	codeGuarantee            = 32
 	codeSeal                 = 33
 	codeTransaction          = 34
 	codeCollection           = 35
 	codeExecutionResult      = 36
-	codeExecutionReceiptMeta = 36
 	codeResultApproval       = 37
 	codeChunk                = 38
+	codeExecutionReceiptMeta = 39 // NOTE: prior to Mainnet25, this erroneously had the same value as codeExecutionResult (36)
```

Comment:
Personally I don't think we should re-use key prefixes until we need to, which could very easily be years away. 

There is a benefit to re-using key prefixes only once we are running out of keyspace. 

By contrast, although it's possible re-using key prefixes is harmless, I think there is a real possibility it has an adverse impact on some users -- for example the Flow Diver team has recently been indexing data from old sporks. I can imagine how a practice of re-using rather than retiring key prefixes could make that sort of thing more challenging.

Reply:
👍